roles/edpm_nova/meta/argument_specs.yml

```diff
@@ -25,10 +25,14 @@ argument_specs:
         description: |
           The path to the directory where the nova config files
           will be rendered on the compute node.
-      edpm_nova_tls_certs_enabled:
+      edpm_nova_live_migration_tls:
```

Comment:
could argue edpm_nova and edpm_libvirt are too tighlty coupled to be separate roles but for now I guess we need to do something like this